### PR TITLE
FE tweaks

### PR DESCRIPTION
**Pull request by projectmushroom**
- [x] mobile font size adjustments
- [x] added user agent checking
- [x] mobile image handling, smaller dimensions and jpeg quality adjustments
- [x] smaller condensed mobile breadcrumbs
- [x] minify styles and scripts

### CODE REVIEW

File: girleffect/templates/blocks/carousel_item_block.html [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
+{% if desktop %}
+    {% image value.image fill-1600x800 as carousel_image %}
+{% else %}
+    {% image value.image fill-800x400 jpegquality-60 as carousel_image %}
+{% endif %}
+
 <div class="carousel__block-item">
     <div class="carousel__panel-desktop">
-        {% image value.image fill-1600x800  %}
+        <img alt="carousel_image.alt" src="{{ carousel_image.url }}">
```
Comment (Mitso) [inline girleffect/templates/blocks/carousel_item_block.html]: Small comment: Does the markup for desktop and mobile not need to be shown on User agent if statement check as well?

File: girleffect/static_src/site/patterns/01-molecules/hero/_hero.scss [hunk 2]
```diff
@@ -12,12 +12,14 @@
...
-          position: relative;
-          padding-bottom: 50.632911392405063%; /* ratio of image height to width */
-          height: 0;
-          overflow: hidden;
+    &--article-image {
+        margin-top: 1rem;
+
+        @include media-query(tablet-portrait) {
+            position: relative;
+            padding-bottom: 50.632911392405063%; /* ratio of image height to width */
```
Comment (Mitso) [inline girleffect/static_src/site/patterns/01-molecules/hero/_hero.scss]: Can this not be rounded of?

File: girleffect/articles/templates/includes/articles_section.html [hunk 3]
```diff
@@ -12,9 +12,15 @@
         {% include "includes/article_card.html" with article=page.featured_article featured=True %}
     {% endif %}
     <div class="article-container grid grid--three wrapper wrapper--mid">
+      {% if request.user_agent.is_pc or request.user_agent.is_tablet %}
         {% for article in page.articles %}
```
Comment (Mitso) [inline girleffect/articles/templates/includes/articles_section.html]: How is the performance of including statement checks inside {% for article in page.articles %}  loop?

File: girleffect/static_src/site/patterns/01-molecules/hero/_hero.scss (hunk 2 above)
Comment (projectmushroom) [reply girleffect/static_src/site/patterns/01-molecules/hero/_hero.scss]: ahhh but don't you like accuracy?? haha NP I'll round it off

File: girleffect/articles/templates/includes/articles_section.html (hunk 3 above)
Comment (projectmushroom) [reply girleffect/articles/templates/includes/articles_section.html]: its pretty bad in the loops, thats why I kept the user agent check outside and passing only `desktop`

File: girleffect/templates/blocks/carousel_item_block.html (hunk 1 above)
Comment (projectmushroom) [reply girleffect/templates/blocks/carousel_item_block.html]: this template is actually sitting in a loop, for better performance I'm getting a `desktop` value from the parent template, without the user agent check

**Pull request merged**